native/src/routes/Pois.tsx

```diff
@@ -77,20 +82,20 @@ const Pois = ({
   const { t } = useTranslation('pois')
   const theme = useTheme()
   const [selectedFeature, setSelectedFeature] = useState<Feature<Point> | null>(null)
-  const [userLocation, setUserLocation] = useState<number[] | undefined>(undefined)
   const [featureLocations, setFeatureLocations] = useState<Feature<Point>[]>(prepareFeatureLocations(pois))
+  const { location, requestAndDetermineLocation } = useLocation(path === null)
```

Comment:
Why do we only want to use a settings listener if the path is null?

Reply:
if path is null we are in the pois detail view, where we definitely don't want to use the listener. But according to our designs we might want to show the distance there.